### PR TITLE
RTL major refactor

### PR DESCRIPTION
**Pull request by yehudab**
Fixes: #4261 
Changes include:

1. Use [CSS Logical Properties and Values](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) in the SCSS sources
2. Add `dir="rtl"` or `dir="ltr"` to `<html>` tag according to the browser language
3. Use [PostCSS](https://github.com/postcss) tools to generate separate CSS files for .rtl and .ltr modes, and serve the correct file according to the page direction. This was done as a (hopefully) temp measure until CSS Logical becomes more supported in all major browsers.
4. Other small fixes, like correct position of the `delete-shot-button` dialog, or correct position for the screenshots pagination.

### CODE REVIEW

File: static/css/partials/_animations.scss [hunk 1]
```diff
@@ -70,7 +70,7 @@
 @keyframes slide-left {
   0% {
     opacity: 0;
-    transform: translate3d(160px, 0, 0);
+    transform: translate3d(160px, 0, 0); // FIXME: RTL
```
Comment (ianb) [inline static/css/partials/_animations.scss]: Did you intend to leave the FIXME here?

File: server/src/pages/shotindex/view.js [hunk 2]
```diff
@@ -107,6 +107,13 @@ class Body extends React.Component {
     const nextPageNumber = this.props.pageNumber - 0 + 1;
     const nextClasses = ["shots-page-nav"].concat(!hasNext && "disabled").join(" ");
     const hidden = totalPages < 2;
+    let arrowheadPrev = this.props.staticLink("/static/img/arrowhead-left-16.svg");
+    let arrowheadNext = this.props.staticLink("/static/img/arrowhead-right-16.svg");
+    if (document.dir === "rtl") {
+      const save = arrowheadNext;
+      arrowheadNext = arrowheadPrev;
+      arrowheadPrev = save;
```
Comment (ianb) [inline server/src/pages/shotindex/view.js]: Doesn't really matter, but you can use fancy JS here, like `[arrowheadNext, arrowheadPrev] = [arrowheadPrev, arrowheadNext]`

File: server/src/l10n.js [hunk 3]
```diff
@@ -84,6 +84,14 @@ exports.getStrings = function(locales) {
   return strings;
 };
 
+exports.getIsRtl = function(locales) {
+  const availableLocales = exports.getUserLocales(locales);
+  if (availableLocales && availableLocales.length > 0) {
+    return /^(he|ar|fa)$/.test(availableLocales[0]);
```
[the remark this answers is not in the document]
Comment (yehudab) [reply server/src/l10n.js]: Added "ur" to RTL languages.

**Comment by ianb** [comment]
I'm not sure I resolved those package-lock conflicts correctly, but who can ever know? I'll try to regenerate them after this lands.

**Pull request merged**